dev/README.md

```diff
@@ -388,8 +341,11 @@ with INSTALL instructions.
 Public keys are available at:
 https://dist.apache.org/repos/dist/release/airflow/KEYS
 
-Only votes from PMC members are binding, but members of the community are
-encouraged to test the release and vote with "(non-binding)".
+Only votes from PMC members are binding, but the release manager should encourage members of the community
+to test the release and vote with "(non-binding)".
+
+The test procedure for PMCs and Contributors who would like to test this RC are described in
```

Comment:
@ashb @kaxil @mik-laj -> also here, the link to test procedure for PMCs and contributors is added